distributionviewer/core/static/js/app/components/views/example-chart.js

```diff
@@ -29,6 +29,7 @@ export class ExampleChart extends React.Component {
       { x: 100, y: 86 },
     ];
 
+    /* eslint-disable camelcase */
```

Comment:
Stuff like this makes me dislike ESLint but let's try it and see if it gets too exception heavy.